namerd/iface/interpreter-thrift/src/main/scala/io/buoyant/namerd/iface/ThriftNamerInterface.scala

```diff
@@ -335,11 +335,16 @@ class ThriftNamerInterface(
       }
     }
     AddrObserver(resolution, stamper)
-  })
+  }
+  private[this] val addrCache = new ObserverCache[Path, Option[Addr.Bound]](
+    activeCapacity = 100,
+    inactiveCapacity = 10,
+    stats = stats,
+    mkObserver = observeAddr
```

Comment:
There are two gauges (one for active size, one for inactive size) in the same scope.